packages/network-of-terms-catalog/catalog/datasets/abr.jsonld

```diff
@@ -44,7 +44,8 @@
             "actionApplication": {
               "@id": "https://reconciliation-api.github.io/specs/latest/",
               "@type": "SoftwareApplication"
-            }
+            },
+            "urlTemplate": "https://termennetwerk-api.netwerkdigitaalerfgoed.nl/reconcile/{distribution}"
```

Comment:
I’m lazy so using a template here that’s the same for all reconciliation sources: only the distribution IRI is different.